Lib/fontTools/__main__.py

```diff
@@ -20,6 +18,10 @@ def main(args=None):
 	# can be added.  Should we just try importing the fonttools
 	# module first and try without if it fails?
 
+	if len(sys.argv) < 2:
+		sys.argv.append("help")
+	if sys.argv[1] == "-h" or sys.argv[1] == "--help":
+		sys.argv[1] = "help"
```

Comment:
why making a `help` package with `__init__.py` and `__main__.py`? Can't we do with a single `help.py` module instead? I don't think we plan to add additional sub-modules to this.

Reply:
This is leveraging the fact that the `fonttools` binary calls `verb.__main__`.

Reply:
No, It runs any module as if it is called `__main__`, but works on standalone modules too. Try

Reply:
`ImportError: No module named fontTools.help.__main__; 'fontTools.help' is a package and cannot be directly executed`

Reply:
Exactly! Don't have a package folder, just a fontTools/help.py module

Reply:
Got it. Here we go.